tests/api_workflow/test_api_workflow_compute_worker.py

```diff
@@ -43,6 +43,18 @@
     SelectionStrategyType,
     TagData,
 )
+from lightly.openapi_generated.swagger_client.models.selection_config_v3 import (
+    SelectionConfigV3,
+)
```

Comment:
Sorry, I forgot to mention this, but we usually import the swagger generated classes directly from `swagger_client.models` like this:
```
from lightly.openapi_generated.swagger_client.models import (
    SelectionConfigV3,
    SelectionConfigV3Entry,
    ...
)

It makes the code a bit easier to read, the drawback is that the VSCode auto-import function does not use this format.